src/internal/git/push.go

```diff
@@ -98,23 +128,23 @@ func push(localPath, tunnelUrl string, spinner *message.Spinner) error {
 		return fmt.Errorf("unable to remove unused git refs from the repo: %w", err)
 	}
 
-	// Fetch remote offline refs in case of old update or if multiple refs are specified in one package
-	fetchOptions := &git.FetchOptions{
-		RemoteName: offlineRemoteName,
-		Auth:       &gitCred,
-		RefSpecs: []goConfig.RefSpec{
-			"refs/heads/*:refs/heads/*",
-			onlineRemoteRefPrefix + "*:refs/heads/*",
-			"refs/tags/*:refs/tags/*",
-		},
-	}
-
-	err = repo.Fetch(fetchOptions)
-	if errors.Is(err, transport.ErrRepositoryNotFound) {
-		message.Debugf("Repo not yet available offline, skipping fetch")
-	} else if err != nil {
-		return fmt.Errorf("unable to fetch remote cleanly prior to push: %w", err)
-	}
+	// // Fetch remote offline refs in case of old update or if multiple refs are specified in one package
```

Comment:
@Racer159 Can I have some extra eyes / help debugging this later? 

Keep this in was failing the push when two separate refs existed in for the same repo but commenting it out seemed to work fine.

Reply:
Thank you! It's not intuitive to me why getting the error `git.ErrForceNeeded` while fetching doesn't lead to any issues while pushing but 🤷 

And I agree on logging the wildcard error as a warning and attempting to push. I'll do that.